src/promql/src/planner.rs

```diff
@@ -2014,57 +2049,61 @@ mod test {
     use super::*;
 
     async fn build_test_table_provider(
-        table_name: String,
+        table_names: &[(String, String)],
         num_tag: usize,
         num_field: usize,
     ) -> DfTableSourceProvider {
-        let mut columns = vec![];
-        for i in 0..num_tag {
-            columns.push(ColumnSchema::new(
-                format!("tag_{i}"),
-                ConcreteDataType::string_datatype(),
-                false,
-            ));
-        }
-        columns.push(
-            ColumnSchema::new(
-                "timestamp".to_string(),
-                ConcreteDataType::timestamp_millisecond_datatype(),
-                false,
-            )
-            .with_time_index(true),
-        );
-        for i in 0..num_field {
-            columns.push(ColumnSchema::new(
-                format!("field_{i}"),
-                ConcreteDataType::float64_datatype(),
-                true,
-            ));
-        }
-        let schema = Arc::new(Schema::new(columns));
-        let table_meta = TableMetaBuilder::default()
-            .schema(schema)
-            .primary_key_indices((0..num_tag).collect())
-            .value_indices((num_tag + 1..num_tag + 1 + num_field).collect())
-            .next_column_id(1024)
-            .build()
-            .unwrap();
-        let table_info = TableInfoBuilder::default()
-            .name(&table_name)
-            .meta(table_meta)
-            .build()
-            .unwrap();
-        let table = EmptyTable::from_table_info(&table_info);
         let catalog_list = MemoryCatalogManager::with_default_setup();
-        assert!(catalog_list
-            .register_table_sync(RegisterTableRequest {
-                catalog: DEFAULT_CATALOG_NAME.to_string(),
-                schema: DEFAULT_SCHEMA_NAME.to_string(),
-                table_name,
-                table_id: 1024,
-                table,
-            })
-            .is_ok());
+        for (schema_name, table_name) in table_names {
```

Comment:
maybe a better variable name (not `table_names`) is needed to include schema meaning?

Reply:
updated to table_name_tuples